src/main/java/entities/User.java

```diff
@@ -105,11 +105,19 @@ public void setEmail(String email) {
     }
 
     public String getRole() {
-        return UserRole.USER.toString();
+        if (role == null) {
+            return this.role = UserRole.USER.name();
+        } else {
+            return this.role = role;
+        }
     }
 
     public void setRole(String role) {
-        this.role = role;
+        if (role == null) {
+            this.role = UserRole.USER.name();
+        } else {
+            this.role = role;
+        }
```

Comment:
What if the setter wasn't involved? Then we set null to DB?

What if `role` will be `""` - empty string? Or even `"betman"` :)